loginCore/src/main/java/com/hamdan/forzenbook/login/core/data/repository/LoginRepository.kt

```diff
@@ -3,5 +3,5 @@ package com.hamdan.forzenbook.login.core.data.repository
 interface LoginRepository {
     suspend fun requestValidation(email: String)
 
-    suspend fun getToken(email: String? = null, code: String? = null): String?
+    suspend fun getToken(email: String? = null, code: String? = null)
```

Comment:
Not returning string any more?

Reply:
no, I realized I could just get the token there and save it there as well useing the application context to put it into the shared preference